### ISSUE TITLE
All computer / Software are empety OCS 2.9.2 Fresh install

### ISSUE THREAD

**Issue by argj** [OCSInventory-NG/OCSInventory-Server#381]
*It's hard to solve a problem when important details are missing, that why we added this template, to help you and us.*

### General informations
Operating system : Centos 8

### Server informations
Perl version :  This is perl 5, version 26, subversion 3 (v5.26.3) built for x86_64-linux-thread-multi
Mysql / Mariadb / Percona version :  MysqlServer Server version: 8.0.26 Source distribution

### OCS Inventory informations
Ocs server version :  2.9.2

### Problem's description
Hi i have a new installation from 0 server and OCS, when i go to show all computers or software the page is blank or empety.
i see data on the database select * from software i seed all information for example.

And on the inspection i see a error 5005 in this URL

http://SERVER.LOCAL/ocsreports/ajax.php?function=visu_search&fields=HARDWARE-LASTCOME&comp=tall&values=03/17/2022%2023:26&values2=windows&type_field=&no_header=true&no_footer=true

Error: 500 Internal Server Error

**Comment by charleneauger** [comment]
Hi @argj ,

The all software page is populated by crontab.
In /usr/share/ocsinventory-reports/ocsreports/crontab run :  `php cron_all_software.php`

Best regards,
Charlene
